### ISSUE TITLE
Please update your instructions

### ISSUE THREAD

**Issue by ivose** [OfficeDev/office-js-docs-pr#3655]
https://learn.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-custom-functions-quickstart?tabs=excel-windows

Here is telling `yo office` And the second step doesn't work:
`Choose a project type: Excel Custom Functions Add-in project` . In my computer it provides no option `Excel Custom Functions Add-in project`. Trying alternatively `Choose a project type="Office Add-in Task Pane project"`, `script type="javascript"`, `name="My custom functions add-in"`, `officeClientApp="Excel"`. Installing.. .
`cd "My office add-in"`, `npm run build`.

And `npm run start:web -- --document https://<myTenantUser>.sharepoint.com/:x:/s/<mysite>/<myExcelUrl>?e=8...g`

After it i upload my own test addins and try excel function `=CONTOSO.GETSTARCOUNT("OfficeDev", "Excel-Custom-Functions")` on b1-cell. It doesn't work.

Commonly please check all of tutorials, because big part of them are obsolete and almost impossible to learn from them.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4bac6469-78a6-dae9-e6c4-497c953a1576
* Version Independent ID: d860aff2-60ad-c008-2d10-b046f3198f44
* Content: [Custom functions quick start - Office Add-ins](https://learn.microsoft.com/en-us/office/dev/add-ins/quickstarts/excel-custom-functions-quickstart?tabs=excel-windows)
* Content Source: [docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/main/docs/quickstarts/excel-custom-functions-quickstart.md)
* Product: **excel**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thanks for reporting this @ivose. I'm sorry to hear you're having issues with the quickstart.

@alison-mk, could you please investigate?

**Comment by alison-mk** [comment]
Hi @ivose, thank you for submitting an issue. We recently updated and renamed the custom functions projects in Yo Office. If you are using Excel on Windows, choose the project type `Excel Custom Functions using a Shared Runtime` and then follow the existing directions. Here is a screenshot: 

![image](https://user-images.githubusercontent.com/7977793/192435118-d73760e5-1e52-4170-8259-36404124e030.png)

I will update the custom functions quick start and tutorial with the new guidance in PR #3656. This issue will close automatically when that PR is merged.

If you are using Excel on Mac, the Yo Office custom functions projects may not work at this time. I have encountered some bugs with the Mac project that I need to investigate further internally. 

Alison
